src/virtual_dom/vtag.rs

```diff
@@ -375,9 +383,12 @@ impl<COMP: Component> VDiff for VTag<COMP> {
         match reform {
             Reform::Keep => {}
             Reform::Before(before) => {
-                let element = document()
-                    .create_element(&self.tag)
-                    .expect("can't create element for vtag");
+                let element = if let Some(ns) = &self.ns {
+                    document().create_element_ns(&ns, &self.tag)
```

Comment:
I was trying to type in svg notes in yew, and was wondering why it showed in the DOM when inspecting the element, but was not rendered. It turns out, you need to use `document.createElementNS` in javascript in so that the svg document fragment will be valid. 

https://developer.mozilla.org/en-US/docs/Web/API/Document/createElementNS